WriterDesk1/src/pages/Document.js

```diff
@@ -23,6 +23,26 @@ import { TrackingContext } from '@vrbo/react-event-tracking';
 // authentication
 import { authHeader } from "../helpers/auth-header";
 
+
+/**
+ * Function to display the file that is clicked on. Use memo to prevent re-rendering document.
+ * @property {String} path - Path to the current file.
+ * @property {String} type - File extension of the current file.
+ * @property {String} fileId - ID of the current file.
+ * @property {String} fileName - Name of the current file.
+ * @returns AllPagesPDFViewer component
```

Comment:
Vgm ben je de memo(props) argument vergeten? Of is dat iets anders?(alsin, is dat geen argument?)